README.md

```diff
@@ -44,6 +44,14 @@ Structures-Wiz is a JavaScript based npm package for using awesome data structur
 		    - [clear](#st-clear)
 		    - [print](#st-print)
     - [Queue](#queues) :soon:
+    - [Singly LinkedList](#singly-linkedlist)
```

Comment:
Change to vanilla LinkedList and add a provision for instantiation of Singly as well as Doubly